include/deal.II/lac/solver.h

```diff
@@ -283,16 +284,19 @@ class Vector;
  *   SolverControl           solver_control (1000, 1e-12);
  *   SolverCG<>              solver (solver_control);
  *
- *   solver.connect (std::bind (&Step3::write_intermediate_solution,
- *                              this,
- *                              std::placeholders::_1,
- *                              std::placeholders::_2,
- *                              std::placeholders::_3));
+ *   solver.connect ([this](const unsigned int iteration,
+ *                          const double check_value,
+ *                          const Vector<double> *current_iterate)
+ *                {
```

Comment:
the indentation of these braces here doesn't match the `[` above nor the `}` below.